scripts/docker-build.sh

```diff
@@ -23,7 +23,7 @@ if [[ -z $APP_VERSION ]] && [[ -z $APP_VERSION_CODE ]]; then
   exit 2
 fi
 
-apt update && apt install -y zip bc cmake ninja-build jq
+apt update && apt install -y zip bc cmake ninja-build jq ; apt install qt5-default
```

Comment:
```suggestion
apt update && apt install -y zip bc cmake ninja-build jq qt5-default
```
Why not like this?

Reply:
Because I didn't looked well. But it fails anyway. What about putting it into translation.yml?